src/systems/fem_context.C

```diff
@@ -250,8 +259,16 @@ void FEMContext::some_interior_gradient(unsigned int var, unsigned int qp, Outpu
 
 #ifdef LIBMESH_ENABLE_SECOND_DERIVATIVES
 template<typename OutputType,
+         void (FEMContext::*fe_getter)
+           (unsigned int,
+            FEGenericBase
+              <typename TensorTools::MakeReal
+                <typename TensorTools::DecrementRank
+                  <typename TensorTools::DecrementRank
+                    <OutputType>::type>::type>::type> *&)
```

Comment:
Pushed one more commit to the [branch](https://github.com/jwpeterson/libmesh/tree/fem_context_refactor) which makes it compile.  The net lines are the same, but I think this follows the DRY principle a bit better.